responses/cs/HassTurnOff.yaml

```diff
@@ -7,6 +7,6 @@ responses:
       fans_area: "Větráky v {{ slots.area }} byly vypnuty"
       fan: "Větrák {{ slots.name }} vypnut"
       light: "Světlo {{ slots.name }} bylo zhasnuto"
-      cover: "{{ slots.name }} zavřeno"
+      cover: "Zavřeno"
```

Comment:
Is there a reason for this change? Concept is that assistant will tell which device's status he changed.

Reply:
I've changed it because:
- The same is in English
- As a user you see the response just after the request so you know what you asked
- And also it was much easier to make tests work...

Reply:
You have to write a lot more tests when you want to write tests with inflection for entities names :(

Reply:
I see, still that can bring troubles when assistant does not understand you correctly. If you ask to turn on a light in bedroom and it turns on light in bathroom, you know what happened. Otherwise it just says "done" and if nothing happened you need to check whole house on what is wrong.

Reply:
Force pushed.